### ISSUE TITLE
Fix codecov missing base coverage

### ISSUE THREAD

**Issue by hisabimbola** [commercetools/nodejs#179]
### Description

```js

```

**Comment by hisabimbola** [comment]
I think this has been fixed now.

**Comment by hisabimbola** [comment]
Just for the record.

The reason it was failing was because when we commit in travis to release a package, `lerna-semantic-release` adds the `[skip-ci]` flag and that skips ci in travis for that commit, hence preventing a codecov report. Since that is the latest commit on master, a PR will try to compare with that commit and since there is no report, we get the _Missing base report_ error from codecov.

This was fixed internally in codecov so that it will look for a base report to compare with if no report was found with the latest commit on master. For some weird reason that stopped working for us and even after we merge a PR, we were still getting the _Missing base report_ error from codecov.

The error occurred down several PRs and subsequent commits on master were failing due to several reasons including less coverage than what was on master.

The simple fix I did was to manually push a report to the latest commit on master and PRs now have a report to compare with. Also I added a commit with the `[skip ci]` flag and codecov was able to look after the latest commit to the previous commit with a report.

I hope this fix is permanent :crossed_fingers:
